docs: add english documentation for test account (#777)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/anti-addiction/features.mdx

Keep in mind that when keeping track of payment restrictions, the amount spent by a minor on all the games provided by the same business will be accumulated.
For example, if a business has two games, and a minor over the age of 8 but under the age of 16 has already spent 100 CNY in one of the games, they can only spend no more than 100 CNY in the other game.

## Test Account
There are two types of scenarios for using test account:

- The first is when the game is in development and testing, and the test account is needed to test the real name authentication anti-obsession feature.
- The second is when you are applying for the game version number.

> Real-name authentication anti-sedentary provided by the test account is strongly dependent on the TapTap login, in accordance with the requirements of the State Press and Publication Administration and therefore provides 30 accounts, including:

Reviewer: ```suggestion
The test accounts provided by the anti-addiction service are strongly dependent on TapTap Login. 30 test account are provided in accordance with the requirements of the State Press and Publication Administration. The accounts include:
```

> - One group of empty accounts without real names, totalling 3;

Reviewer: ```suggestion
- One group of empty accounts without real-name verification, with a total of 9 accounts;
```

> - Two groups of minor accounts, 18 in total, each containing 3 each of high, medium and low (16 years of age and under 18 years of age, 8 years of age and under 16 years of age, and 8 years of age and under 8 years of age);

Reviewer: ```suggestion
- Two groups of minor accounts, 18 in total, each containing 3 each of high, medium, and low ages (above 16 and under 18, above 8 and under 16, and under 8);
```

- One group of adult accounts (over 18 years old), with a total of 9 accounts.

> According to the usage scenario of the test account, the usage mode of the test account is divided into two types: **Regular Mode** and **Test Mode**.

Reviewer: ```suggestion
According to the usage scenario of test accounts, the usage mode of test accounts is divided into two types: **Regular Mode** and **Test Mode**.
```

### Regular Mode

> The status and attributes of the test account cannot be changed in the regular mode, which is applicable to the scenario of submitting the game to the moderation office for review and applying for a version number.

Reviewer: ```suggestion
The status and attributes of the test accounts cannot be changed in the regular mode, which is applicable to the scenario of submitting the game for review and applying for an ISBN.
```

### Test Mode

> To facilitate developers to test the real-name authentication anti-addiction function for underage players, a test mode is introduced, under which the status and attributes of the test account can be changed. For example, for the underage test account provided, you can change whether the current time is playable or not, and the amount of money that has been recharged, so that the developer does not need to wait for the underage to comply with the time period of the game to test the anti-addiction function. Test accounts can be viewed by going to **TapTap Developer Centre > Your Games > Game Services > Compliance Certification > Test Accounts** in order.

Reviewer: ```suggestion
To facilitate developers to test the real-name verification and anti-addiction functions for underage players, the test mode is introduced, under which the status and attributes of the test accounts can be changed. For example, for the underage test accounts provided, you can change whether the user is playable or not at the moment, and the amount of money that has been spent, so that the developer does not need to wait for the time when underage players are allowed to play to test the anti-addiction function. Test accounts can be viewed by going to **TapTap Developer Centre > Your Games > Game Services > Compliance Certification > Test Accounts**.
```
